addon/data/selector-ui.js

```diff
@@ -31,15 +31,31 @@ const ui = (function () { // eslint-disable-line no-unused-vars
   /** Represents the shadow overlay that covers the whole page */
   let WholePageOverlay = exports.WholePageOverlay = {
 
-    display: function () {
+    display: function (callbacks) {
       if (! this.overlayEl) {
         this.overlayEl = document.createElement("div");
+        this.overlayEl.className = "pageshot-preview-overlay";
         let instructions = document.createElement("div");
         instructions.className = "pageshot-preview-instructions";
-        instructions.textContent = "Click an element to select, drag a region to select, or press ESC to cancel.";
+        instructions.textContent = "Drag or click on the page to select a region. Press ESC to cancel.";
         this.overlayEl.appendChild(instructions);
+        if (callbacks && callbacks.onOpenMyShots !== undefined) {
```

Comment:
Why is there a conditional here?  Is there a case where this is being called without the callbacks?